tests/signal-cast-registerer.cpp

```diff
@@ -146,7 +146,7 @@ BOOST_AUTO_TEST_CASE(custom_vector_registerer) {
   } catch (const ExceptionSignal &aes) {
     res = (aes.getCode() == ExceptionSignal::GENERIC);
   }
-  //  BOOST_CHECK(res);
+  BOOST_CHECK(res); // Uncomment this line to resolve the unused-but-set-variable warning to have a stable release in ros buildfarm
```

Comment:
This was commented because the test didn't pass:
```
 1/18 Test  #1: signal-cast-registerer ...........***Failed    0.02 sec
Running 3 test cases...
/home/gsaurel/local/stack-of-tasks/dynamic-graph/tests/signal-cast-registerer.cpp(149): error: in "custom_vector_registerer": check res has failed
Test passed : ss[0] != "["Test passed : ss[1] != %iTest passed : ss[2] != "]"Test passed : ss[3] != "("Test passed : ss[-1] != ")"Test passed : ss[0] != "["Test passed : ss[1] != %iTest passed : ss[2] != ","Test passed : ss[3] != %iTest passed : ss[4] != "]"Test passed : ss[5] != "("Test passed : ss[6] != "("ss[6+n] != ")"Test passed : ss[5] != ")"Test passed : ss[5] != ")"Test passed : ss[5] != ")" and ignore ","
*** 1 failure is detected in the test module "signal_cast_registerer"
```

So ideally, we should fix this test, or remove it if it is useless.  In any case, we can't merge this PR as it.

@olivier-stasse : you introduced this in https://github.com/stack-of-tasks/dynamic-graph/commit/d5039779137e75b0c9d6641023e5a5adf11c7599 ; could you explain what you wanted to do, and what is left to do ?

Reply:
This is removed on branch cpp11. To make merging easy, I suggest not to change it.

Reply:
Do you know an alternative to fix this warning :  unused-but-set-variable warning for the res value?

Reply:
After double-thinking, the file is removed so the merge conflict will be easily solved. You can remove the variable `res`.

Reply:
Ok so I need to remove all lines from 142 to 149?

Reply:
Yes, you can do that.

Reply:
Ok thank you ^^